packages/mui-material/src/styles/createTheme.test.js

```diff
@@ -222,4 +223,35 @@ describe('createTheme', () => {
     );
     expect(container.firstChild).toHaveComputedStyle({ fontFamily: 'cursive' });
   });
+
+  it('should apply the correct styles via sx prop if theme values are 0', function test() {
+    const isJSDOM = /jsdom/.test(window.navigator.userAgent);
+
+    if (isJSDOM) {
+      this.skip();
+    }
+
+    const theme = createTheme({
+      shape: {
+        borderRadius: 0,
+      },
+      spacing: 0,
```

Comment:
I think let's test only the border radius. `spacing: 0` sounds impossible to me.

Reply:
Okay